### PR TITLE
Support reading from stdin for longer input data.

### LINKED ISSUE #3: Change NBFS to read input from stdin
NBFS fails to decode big payloads because of command line restriction

### PR DESCRIPTION
**Pull request by JarLob**
Resolves #3

### CODE REVIEW

File: src/NBFS/NBFS.cs [hunk 1]
```diff
@@ -27,8 +27,6 @@
  */
 
 using System;
-using System.Collections.Generic;
```
Comment (antnks) [inline src/NBFS/NBFS.cs:30]: 👍

**Review by antnks — APPROVED** [review]
Not good enough

**Comment by antnks** [comment]
Smoke test failed:
```
$ dotnet run
Usage: NBFS [encode|decode] Base64Data

NOTE: All output, including exceptions, will be returned as a Base64 string.

Unhandled Exception: System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at NBFSNetConsole.Main(String[] argv) in /home/dotnet/git/wcf-deserializer/src/NBFS/NBFS.cs:line 45
```

**Review by antnks — CHANGES_REQUESTED** [review]
$ dotnet run
Usage: NBFS [encode|decode] Base64Data

NOTE: All output, including exceptions, will be returned as a Base64 string.

Unhandled Exception: System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at NBFSNetConsole.Main(String[] argv) in /home/dotnet/git/wcf-deserializer/src/NBFS/NBFS.cs:line 45

**Comment by JarLob** [comment]
Main workflow was passing 😋 

> Smoke test failed:

**Review by antnks — APPROVED** [review]

**Pull request merged**